Lib/asyncio/base_events.py

```diff
@@ -295,7 +295,7 @@ def _make_socket_transport(self, sock, protocol, waiter=None, *,
 
     def _make_ssl_transport(self, rawsock, protocol, sslcontext, waiter=None,
                             *, server_side=False, server_hostname=None,
-                            extra=None, server=None):
+                            extra=None, server=None, ssl_handshake_timeout=10.0):
```

Comment:
Add `SSL_HANDSHAKE_TIMEOUT = 10.0` to `constants.py` and reuse the constant everywhere.

Reply:
I mean everywhere you have `10.0` constant.